Description:
Attach the boinc process to boincmgr so that nothing runs in the background unintentionally
Launch the advanced GUI to avoid a broken simple GUI (https://github.com/BOINC/boinc/issues/3415)

diff a/clientgui/BOINCGUIApp.cpp b/clientgui/BOINCGUIApp.cpp
--- a/clientgui/BOINCGUIApp.cpp
+++ b/clientgui/BOINCGUIApp.cpp
@@ -186,10 +186,10 @@ bool CBOINCGUIApp::OnInit()
     // Restore Application State
     m_pConfig->SetPath(wxT("/"));
-    m_pConfig->Read(wxT("AutomaticallyShutdownClient"), &m_iShutdownCoreClient, 0L);
+    m_pConfig->Read(wxT("AutomaticallyShutdownClient"), &m_iShutdownCoreClient, 1L);
     m_pConfig->Read(wxT("DisplayShutdownClientDialog"), &m_iDisplayExitDialog, 1L);
     m_pConfig->Read(wxT("DisplayShutdownConnectedClientDialog"), &m_iDisplayShutdownConnectedClientDialog, 1L);
     m_pConfig->Read(wxT("DisableAutoStart"), &m_iBOINCMGRDisableAutoStart, 0L);
     m_pConfig->Read(wxT("LanguageISO"), &m_strISOLanguageCode, wxT(""));
-    m_pConfig->Read(wxT("GUISelection"), &m_iGUISelected, BOINC_SIMPLEGUI);
+    m_pConfig->Read(wxT("GUISelection"), &m_iGUISelected, BOINC_ADVANCEDGUI);
     m_pConfig->Read(wxT("EventLogOpen"), &bOpenEventLog);
     m_pConfig->Read(wxT("RunDaemon"), &m_bRunDaemon, 1L);
